docs/arch/monitor.md

```diff
@@ -0,0 +1,93 @@
+# Monitor
+
+## Contributors
+
+- Sarah THEOULLE
+- Pauline CONTAT
+- Thomas BROINE
+- Baptiste BRONSIN
+
+## Functionalities
+
+- Listening to events from a remote Git repository
+- Recognizing the event type
+- Adapting actions according to the event type and then calling the controller via an external API
+
```

Comment:
When a event that you recognize happen you trigger a pipeline. It could be added to this part.